README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
```

Comment:
It worth to state in the first description that the project is in WIP and request for support.